pkg/gui/presentation/commits.go

```diff
@@ -43,6 +44,7 @@ func GetCommitListDisplayStrings(
 	length int,
 	showGraph bool,
 	bisectInfo *git_commands.BisectInfo,
+	youAreHereStr string, // non-empty only when we are rebasing
```

Comment:
I'm kind of abusing this parameter as both a way to pass in the translated string (`Tr` isn't known in here), and as a flag to say whether we should add the label or not. Hopefully the comment makes this clear enough.